### PR TITLE
Ethereum

### PR DESCRIPTION
**Pull request by enr74**

### CODE REVIEW

**Comment by massigerardi** [comment]
Running io.yope.ethereum.AuthorityTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 16.938 sec <<< FAILURE! - in io.yope.ethereum.AuthorityTest
testCreateAndRunAuthority(io.yope.ethereum.AuthorityTest)  Time elapsed: 16.754 sec  <<< ERROR!
com.googlecode.jsonrpc4j.HttpException: <html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.9.3 (Ubuntu)</center>
</body>
</html>

	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1838)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1439)
	at com.googlecode.jsonrpc4j.JsonRpcHttpClient.invoke(JsonRpcHttpClient.java:158)
	at com.googlecode.jsonrpc4j.ProxyUtil$3.invoke(ProxyUtil.java:242)
	at com.sun.proxy.$Proxy6.eth_compileSolidity(Unknown Source)
	at io.yope.ethereum.services.ContractService.create(ContractService.java:42)
	at io.yope.ethereum.services.EthereumFacade.createContracts(EthereumFacade.java:21)
	at io.yope.ethereum.AuthorityTest.testCreateAndRunAuthority(AuthorityTest.java:29)

File: yope-careers-service/src/main/java/io/yope/careers/visitor/CandidateVisitor.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Override
+    public String getContractFile() {
+        return "CandidateContract.sol";
+    }
+
+    @Override
+    public String getContractContent() {
+        return removeLineBreaks(getContractFile());
```
Comment (massigerardi) [inline yope-careers-service/src/main/java/io/yope/careers/visitor/CandidateVisitor.java]: I could not find the method removeLineBreaks

Comment (enr74) [inline yope-careers-service/src/main/java/io/yope/careers/visitor/CandidateVisitor.java]: It is in: https://github.com/yopeio/ethereum-api/blob/cc5373a03f4a4c1eb97e26e471d4fa3ba1bad6ba/yope-ethereum-model/src/main/java/io/yope/ethereum/utils/EthereumUtil.java#L11

Comment (enr74) [inline yope-careers-service/src/main/java/io/yope/careers/visitor/CandidateVisitor.java]: Unit test should return:
```
21:54:46.087 [main] INFO  io.yope.ethereum.AuthorityTest - receipt: Receipt(transactionHash=0xfcc2bed99075314952277d2b44ae78d151c5560538614b161c18518f4f81971d, contractAddress=0x946d327cef5eb27dd848d19f586bcafb89af44b2, blockHash=0x7245afa6ec68c53158c73ee9a732685e86837728f55e2896e412ce4add0a0818, transactionIndex=0, blockNumber=200, cumulativeGasUsed=85446, gasUsed=85446)
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.468 sec - in io.yope.ethereum.AuthorityTest
```

**Comment by massigerardi** [comment]
it seems I cannot access the server from my ip or the service is not working

**Pull request closed without merging**